.gitignore

```diff
@@ -40,3 +40,6 @@ integration/kubeconfig-*
 
 # direnv's files:
 .envrc
+
+#vscode
```

Comment:
We try not to add environment specific rules here. You can ignore these files for all your repos by having your own `~/.gitignore_global` file ([docs](https://help.github.com/en/github/using-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer)).

Reply:
Thanks! I didn't know that. I was just going to remove it before it ever gets merged. Thanks for the tip. Will do